hack/testing/calculate_performance_thresholds.py

```
@@ -0,0 +1,439 @@
+        
+        print(f"Found {len(summary_files)} test result file(s)")
+        
+        for summary_file in sorted(summary_files):
+            try:
+                with open(summary_file, 'r') as f:
+                    data = yaml.safe_load(f)
+                    if data:
+                        self._extract_metrics(data)
+                        run_count += 1
+                        print(f"  ✓ Loaded {summary_file}")
+            except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)